sycl/source/detail/context_impl.hpp

```diff
@@ -341,5 +341,21 @@ void GetCapabilitiesIntersectionSet(const std::vector<sycl::device> &Devices,
 }
 
 } // namespace detail
+
+// We're under sycl/source and these won't be exported but it's way more
```

Comment:
these are awesome.   I find myself reaching for `getSyclObjImp`, `getAdapter` and `getHandlerRef` on a nearly daily basis.

But how will other team members know to use them?

Reply:
I've updated all the places I found that can benefit from them. Hopefully copy-paste will spread the usage from here.